examples/step_operator_remote_training/run.py

```diff
@@ -68,9 +57,36 @@ def mnist_pipeline(
     evaluator(X_test=X_test, y_test=y_test, model=model)
 
 
-pipeline = mnist_pipeline(
-    importer=importer(),
-    trainer=trainer(),
-    evaluator=evaluator(),
+@click.command()
+@click.option(
+    "--step_operator",
+    default="sagemaker",
+    type=click.Choice(["sagemaker", "azureml", "vertex"], case_sensitive=True),
+    help="Type of operator, choice from 'azureml', 'sagemaker', or 'vertex'",
```

Comment:
I think this should specify probably that it should be the same as the name of a registered step operator stack component name

Reply:
Makes sense!